src/test/scala/chiselTests/LoadMemoryFromFileSpec.scala

```diff
@@ -0,0 +1,194 @@
+// See LICENSE for license details.
+
+package chiselTests
+
+import java.io.{ByteArrayOutputStream, File, PrintStream}
+
+import chisel3._
+import chisel3.util.experimental.loadMemoryFromFile
+import chisel3.util.log2Ceil
+import firrtl.FirrtlExecutionSuccess
+import firrtl.annotations.MemoryLoadFileType
+import org.scalatest.{FreeSpec, Matchers}
+
+//noinspection TypeAnnotation
```

Comment:
I prefer to prevent IntelliJ from getting to commit to the repo... Is it okay to remove this?

Reply:
Scalastyle comments lgtm.